Sources/TracerUI/Logger/ItemLoggerListPresenter.swift

```diff
@@ -20,10 +20,14 @@ final class ItemLoggerListPresenter: Presenting {
     
     func listenForChanges() {
         TraceUISignals.Logger.itemLogged.listen { loggedItem in
-            self.view.display(loggedItem: loggedItem)
+            DispatchQueue.main.async {
+                self.view.display(loggedItem: loggedItem)
```

Comment:
Looks like the original code could create a retain cycle on self? Can we add that to a capture list?